regression/goto-analyzer-simplify/simplify-multiply-by-zero/test.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+main.c
+"--variable"
```

Comment:
Likewise.

Reply:
There shouldn't be a need for quotes here. Double quotes, in particular, break AppVeyor.